ironic-scripts/runironic-api

```diff
@@ -19,5 +19,12 @@ done
 
 python3 -c 'import os; import sys; import jinja2; sys.stdout.write(jinja2.Template(sys.stdin.read()).render(env=os.environ))' < /etc/httpd-ironic-api.conf.j2 > /etc/httpd/conf.d/ironic.conf
 sed -i "/Listen 80/c\#Listen 80" /etc/httpd/conf/httpd.conf
+
+if [ "$IRONIC_TLS_SETUP" == "true"  && "${RESTART_CONTAINER_CERTIFICATE_UPDATED}" == "true" ]; then
+    inotifywait -m -e delete_self "${IRONIC_CERT_FILE}" | while read file event; do
+      kill $(pgrep httpd)
```

Comment:
This should use `kill -HUP` (for a graceful restart), and I suspect the same is true of the others.

Reply:
I tried to send SIGHUP signal to the ironic-conductor process and I can see that it reloads the config, but it does not re-read the TLS certificate file. So the only way is to restart the container.

Reply:
just a thought, how about ditching apache in favor of nginx? 
(nginx can dynamically load SSL certs)

Reply:
Btw, thanks for the suggestion to use `kill -WINCH`. It is helpful.

Reply:
@s3rj1k Apache server can also do the same, actually. The reason why I restart the httpd is to make it consistent with the other containers